pkg/validation/sessions.go

```diff
@@ -5,8 +5,13 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
```

Comment:
Same note here as before around the `session` confusion. `awssession` might be nice.